layouts/shortcodes/logos__item.html

```diff
@@ -0,0 +1,33 @@
+{{- $lazy := .Get "lazy" | default true -}}
+
+{{ if .Get "link" }}
+    <a href="{{ .Get "link" }}" class="logos__item logos__item--link">
+        <img src="{{ .Get "src" }}"
+            class="logos__logo"
+            width="200"
+            height="auto"
+            {{ if .Get "alt" }}
+                alt="{{ .Get "alt" }}"
+            {{ end }}
+            {{ if (and (ne $lazy false) (ne $lazy "false")) }}
```

Comment:
It would be nice if we could already convert this to a boolean where we set the variable Lazy at the top of this file.

Reply:
Done!